CtrlVAF/CtrlVAF/Events/EventDispatcher.cs

```diff
@@ -31,13 +34,26 @@ public class EventDispatcher : Dispatcher
         /// <param name="command">A command of type TCommand that has inherited from <see cref="Commands.IEventCommand{T}"/></param>
         /// <param name="throwExceptions">Whether or not to stop executing ICommandHandlers upon exceptions and throw the exception</param>
         /// <param name="exceptionHandler">An exception handler to pass along or handle any ICommandHandler exceptions</param>
-        public EventDispatcher()
+        public EventDispatcher(ConfigurableVaultApplicationBase<TConfig> vaultApplication)
         {
+            this.vaultApplication = vaultApplication;
         }
 
         /// <inheritdoc/>
         public override void Dispatch(params ICtrlVAFCommand[] commands)
         {
+            var wrongCommands = commands.ToList().RemoveAll(cmd =>
+               cmd.GetType() != typeof(EventCommand) ||
+               cmd.GetType().BaseType == typeof(EventCommand));
+            if (wrongCommands > 0)
+            {
+                SysUtils.ReportWarningToEventLog(
```

Comment:
This call fails whenever doing test, meaning that multiple tests fail. I think this reporting to the event log can be omitted.

Reply:
That's weird. SysUtils is a helper class from MFiles.VAF and I have no idea why it's failing. But yeah it can be removed with no harm. I have some other locations where SysUtils tries to log erors and infos though, we might want to check if those fail for you too. 😕